src/main/java/seedu/hirehub/model/application/exceptions/DuplicateApplicationException.java

```diff
@@ -0,0 +1,11 @@
+package seedu.hirehub.model.application.exceptions;
+
+/**
+ * Signals that the operation will result in duplicate Persons (Persons are considered duplicates if they have the same
+ * identity).
+ */
```

Comment:
Change comment to "duplicate Applications"

Reply:
corrected